### PR TITLE
Live migration tests - Additions and fixes

### PR DESCRIPTION
**Pull request by enp0s3**
Signed-off-by: Igor Bezukh ibezukh@redhat.com

**What this PR does / why we need it**:
We need to test a live migration case where a Linux VM has guest agent inside. To be more specific, this test was already implemented but labeled incorrectly, so the PR fixes the labeling. In addition, some cosmetic changes were added while working on the PR.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
Test IDs 2653

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by rmohr** [comment]
/lgtm
/approve

Looks good. If tests pass, it can go in. Thanks :+1:

File: tests/migration_test.go [hunk 1]
```diff
@@ -804,14 +855,14 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 				By("Checking that the migrated VirtualMachineInstance console has expected output")
 				expecter, err = tests.ReLoggedInFedoraExpecter(vmi, 60)
 				defer expecter.Close()
-				Expect(err).ToNot(HaveOccurred())
+				Expect(err).ToNot(HaveOccurred(), "Should be able to re-login to the migrated VM")
```
Comment (omeryahud) [inline tests/migration_test.go]: I think "Should stay logged in after migration" makes more sense

Comment (enp0s3) [reply tests/migration_test.go]: Done

**Comment by enp0s3** [comment]
@rmohr @omeryahud I am holding the PR, need to check why tests are failing. On my local server these tests do pass w/o issue

**Pull request merged**